cmd/logger.go

```diff
@@ -17,78 +17,50 @@
 package cmd
 
 import (
+	"encoding/json"
 	"fmt"
-	"io/ioutil"
+	"os"
 	"path"
+	"path/filepath"
 	"runtime"
+	"strconv"
 	"strings"
+	"time"
 
-	"github.com/Sirupsen/logrus"
 	"github.com/minio/mc/pkg/console"
 	"github.com/minio/minio/pkg/errors"
 )
 
 var log = NewLogger()
 
-// LogTarget - interface for log target.
-type LogTarget interface {
-	Fire(entry *logrus.Entry) error
-	String() string
+type logEntry struct {
+	Level   string   `json:"type"`
```

Comment:
1. Done.
2. AB wanted to call it `Type: `, when the log information is printed out in json format.
In console logs though, we do not print a label for `type`, but just the value of it in upper case, like `ERROR` or `FATAL`.

Reply:
`type` json field may confuse consumer by some other meaning (e.g. subsystem/facility) than `level` and `level` is a known name for log classification in logging.  Can you confirm with AB once again?

Reply:
Sure! That is a good point and I remember there was the same question when we discussed it, but we can still discuss it again with AB and modify it if it is decided to be so. 
Could we move on with this PR though, instead of blocking it for this minor topic?
We can always open up a new issue if we decide to rename it.

Reply:
Once we start logging it's an API any change means breaking compatiblity.  So we need to discuss and move forward.

Reply:
Sure. Then, we'll discuss and resolve it when AB is back.